src/wp-includes/rest-api/endpoints/class-wp-rest-edit-site-export-controller.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * REST API: WP_REST_Edit_Site_Export_Controller class
+ *
+ * @package    WordPress
+ * @subpackage REST_API
+ */
+
+/**
+ * Controller which provides REST endpoint for exporting current templates
+ * and template parts.
+ *
+ * @since 5.9.0
+ *
+ * @see WP_REST_Controller
+ */
+class WP_REST_Edit_Site_Export_Controller extends WP_REST_Controller {
+
+	/**
+	 * Constructs the controller.
+	 */
+	public function __construct() {
+		$this->namespace = 'wp-block-editor/v1';
+		$this->rest_base = 'export';
+	}
+
+	/**
+	 * Registers the necessary REST API routes.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base,
+			array(
+				array(
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'export' ),
+					'permission_callback' => array( $this, 'permissions_check' ),
+				),
+			)
+		);
+	}
+
+	/**
+	 * Checks whether a given request has permission to export.
+	 *
+	 * @return WP_Error|bool True if the request has access, or WP_Error object.
+	 */
+	public function permissions_check() {
+		if ( current_user_can( 'edit_theme_options' ) ) {
```

Comment:
I'm also not sure, so I kept the cap that was used in the original code. I think we might need new caps for block theme exports in the long run.

Reply:
Yeah I don't have a solid answer either. As you said, I think `export` is more on the side of site data, whereas this is less about site content but appearance. It feels like it may be a candidate for a dedicated meta capability that maps to both by default?

Reply:
Thanks both. Let's consider this but not treat is as a blocker for getting this patch in. As is, I think the permission check is fit for purpose.